xclim/data/fr.json

```diff
@@ -475,45 +428,39 @@
     "abstract": "Indicateur des conditions nycthermiques de maturation, qui correspond à la valeur moyenne de la température minimale de l'air sur les 30 jours précédant la date de récolte potentielle."
   },
   "DLYFRZTHW": {
-    "long_name": "Cycles de gel-dégel quotidien",
-    "description": "Nombre {freq:m} de jours avec un gel-dégel : Tmax > {thresh_tasmax} et Tmin <= {thresh_tasmin}.",
+    "long_name": "Cycles de gel-dégel",
+    "description": "Nombre {freq:m} de jours avec un gel-dégel quotidien : Tmax > {thresh_tasmax} et Tmin <= {thresh_tasmin}.",
     "title": "Nombre de jours avec un cycle de gel-dégel",
-    "comment": "",
-    "abstract": "Le nombre de jours où Tmax est au-dessus d'un seuil et Tmin sous un seuil, habituellement 0°C pour les deux."
+    "abstract": "Le nombre de jours où Tmax est au-dessus d'un seuil et Tmin au-dessous d'un seuil, habituellement 0 °C pour les deux."
   },
   "COOLING_DEGREE_DAYS": {
-    "long_name": "Degré-jour de réfrigération (Tmoy > {thresh})",
+    "long_name": "Degrés-jours de réfrigération (Tmoy > {thresh})",
```

Comment:
Wouldn't it be "refroidissement" ou "climatisation", instead of "réfrigération"?

Reply:
ClimateData.ca dit "refroidissement" en effet, je voterais pour ça. Cela dit "réfrigération" existe ailleurs.

Reply:
donneesclimatiques.ca dit "climatisation". Je trouve que c'est plus logique puisque le heating_degree_days c'est pour savoir quand tu dois utiliser le chauffage et le cooling_degree_days quand tu dois lancer la climatisation.

Reply:
Selon moi les deux sont défendables, mais je voterais pour "climatisation". En tout cas, "réfrigération" n'est pas le bon mot ici.